src/stores/game.js

```diff
@@ -387,7 +396,7 @@ export const useGameStore = defineStore('game', {
               this.myPNum = res.pNum;
               this.updateGame(res.game);
               this.successfullyJoined({
-                username: res.playerUsername,
+                username: res.username, // was playerUsername but it doesn't exist 
```

Comment:
No comment needed here either.

Reply:
I'm confused, I corrected this... I'm git confused I think